src/types/viewport.cpp

```diff
@@ -355,22 +347,18 @@ bool Viewport::DecrementInBoundsCircular(til::point& pos) const noexcept
 // Arguments:
 // - first- The first coordinate position
 // - second - The second coordinate position
-// - allowEndExclusive - if true, allow the EndExclusive til::point as a valid position.
-//                        Used in accessibility to signify that the exclusive end
-//                        includes the last til::point in a given viewport.
 // Return Value:
 // -  Negative if First is to the left of the Second.
 // -  0 if First and Second are the same coordinate.
 // -  Positive if First is to the right of the Second.
 // -  This is so you can do s_CompareCoords(first, second) <= 0 for "first is left or the same as second".
 //    (the < looks like a left arrow :D)
 // -  The magnitude of the result is the distance between the two coordinates when typing characters into the buffer (left to right, top to bottom)
-#pragma warning(suppress : 4100)
-int Viewport::CompareInBounds(const til::point first, const til::point second, bool allowEndExclusive) const noexcept
+int Viewport::CompareInBounds(const til::point first, const til::point second) const noexcept
 {
     // Assert that our coordinates are within the expected boundaries
-    assert(IsInBounds(first, allowEndExclusive));
-    assert(IsInBounds(second, allowEndExclusive));
+    assert(!IsInBounds(first));
+    assert(!IsInBounds(second));

```

Comment:
this assertion looks like it should fail -- it is the opposite of the comment. Assert that both  first and second are out of bounds??